contents/docs/product-analytics/identify.mdx

```diff
@@ -190,7 +190,9 @@ You can view whether a user can be merged into another user using `alias` when [
 
 ## Anonymous and identified events
 
-If your `person_profile` config option is set to `identified_only`, PostHog only captures identified events and creates a person profile once you call one of the following functions or methods:
+If your `person_profile` config option is set to `identified_only`, PostHog will continue to capture all events but only events which count as identified will be associated with a person profile. All other events will display in your activity tab with this note when you click on the user Id for the person: "No profile associated with this ID".
+
+When using the `identified_only` option, person profiles are only created once you have called one of the following functions or methods:
```

Comment:
```suggestion
If your `person_profile` config option is set to `identified_only`, PostHog continues to capture all events but, for users without a person profile, these will be anonymous events. This means you can't do person-level analysis on them. For example, when you click the person ID in the activity tab, you'll see a note saying "No profile associated with this ID."

When using the `identified_only` option, person profiles are only created once you call one of the following functions or methods:
```

Reply:
@ivanagas Thank you! this is great - I was really struggling to word this one.